.github/workflows/publish.yml

```
@@ -0,0 +1,62 @@
+  checksums:
+    needs:
+      - android
+      - desktop
+      - void
+    permissions:
+      contents: write
+      actions: read
+    uses: ./.github/workflows/release-checksums.yml
+    secrets: inherit
+    with:
+      tagName: ${{ github.event.release.tag_name }}
```

actionlint (1.7.8)

input "tagName" is not defined in "./.github/workflows/release-checksums.yml" reusable workflow. no input is defined

(workflow-call)